test.js

```diff
@@ -77,6 +77,20 @@ test('remarkTextr', async function (t) {
       'yo «there» …\n'
     )
   })
+
+  await t.test('should support syncronous transformer', async function () {
+    assert.equal(
+      String(
+        await remark()
+          .use(remarkTextr, {
+            plugins: [typographicQuotes],
+            options: {locale: 'ru'}
+          })
+          .process('yo "there" \n')
```

Comment:
Should this be `processSync` and without `await`?

Reply:
You are right. I will employ the processSync.

Reply:
Shall I use the `processSync ` for other tests as well, for example for ones don't have options, empty plugin etc.?

Reply:
No, I generally like using `await` and `process`!

Reply:
ok